pkg/db/provider.go

```diff
@@ -3,14 +3,23 @@ package db
 import (
 	"fmt"
 
+	"github.com/ArkeoNetwork/directory/pkg/types"
 	"github.com/pkg/errors"
 )
 
 type ArkeoProvider struct {
 	Entity
 	Pubkey string `db:"pubkey"`
 	Chain  string `db:"chain"`
-	Bond   string `db:"bond"` // this is a DECIMAL type in the db
+	// this is a DECIMAL type in the db
+	Bond                string               `db:"bond"`
+	MetadataURI         string               `db:"metadata_uri"`
+	MetadataNonce       uint64               `db:"metadata_nonce"`
+	Status              types.ProviderStatus `db:"status,text"`
+	MinContractDuration int64                `db:"min_contract_duration"`
+	MaxContractDuration int64                `db:"max_contract_duration"`
+	SubscriptionRate    int64                `db:"subscription_rate"`
+	PayAsYouGoRate      int64                `db:"paygo_rate"`
```

Comment:
should we add coordinates here?

Reply:
unsure, suspect that parts of metadata.json? haven't come across it in any of the event attributes